frontend/packages/local-storage-operator-plugin/src/components/auto-detect-volume/auto-detect-volume.scss

```diff
@@ -6,3 +6,13 @@
 .co-m-nav-title {
   margin-top: 0;
 }
+
+.auto-detect-volume__node-list {
+  @media (min-width: 769px) {
+    max-width: 100% !important;
+  }
+
+  @media (min-width: 1200px) {
+    max-width: 70% !important;
```

Comment:
can we avoid `!important` ?

Reply:
We need to override some media queries in `co-m-pane__form`, so we need to use `!important`